docs/ref/signals.txt

```diff
@@ -303,6 +303,15 @@ Arguments sent with this signal:
 ``using``
     The database alias being used.
 
+``raw``
+
+    .. versionadded:: 6.1
+
+    A boolean; ``True`` if the model is saved exactly as presented
+    (i.e. when loading a :ref:`fixture <fixtures-explanation>`). One should not
+    query/modify other records in the database as the database might not be in
+    a consistent state yet.
```

Comment:
I think for consistency we should add `raw` to the tables below ("the arguments sent ... would be:")

Reply:
Good catch :dart: Added.